design/protocol-changes/protocol-parameters.tex

```diff
@@ -0,0 +1,48 @@
+\section{Protocol parameters}\label{sec:protocol parameters}
+
+The Peras protocol is influenced by various new protocol parameters, also see~\cite{peras-cip}, which we list in \cref{fig:protocol parameters}.
+For explicitness and to avoid confusion, we use a relatively verbose naming scheme.
+The Cardano Ledger allows protocol parameters to be controlled via on-chain governance.
+We indicate whether this flexibility is likely to be useful.
+
+\begin{figure}[h]
+  \centering
+  \begin{tabular}{c c c c c}
+    \toprule
+    Name & Unit & Symbol & Feasible value & Governable? \\
+    \midrule
+    \perasRoundSlots{} & \unit{\slot} & $U$ & \num{90} & $\checkmark$ \\
+    \perasBlockMinSlots{} & \unit{\slot} & $L$ & $T_{\mathrm{CQ}} = \kcg/\asc$ & $\checkmark$ \\
+    \perasBlockMaxSlots{} & \unit{\slot} & n.a. & \numrange{30}{90} & $\checkmark$ \\
```

Comment:
We need at the very least `\perasBlockMaxSlots > \perasBlockMinSlots`. Or maybe is this line suggesting that if `\perasBlockMinSlots = 30`, then `\perasBlockMaxSlots = 90`? That sounds too small, because there is only a 95% chance (with asc = 1/20) that there is a block in 60 slots, so in 5% of the cases we would not be allowed to vote in a round.

Also, maybe it would be worth defining `\perasBlockMaxSlots` not from the first slot of the round but from `\perasBlockMinSlots`. If `s` is the first slot of the current round, then we would be allowed to vote for a block of slot t such that:

    s - \perasBlockMinSlots - \perasBlockMaxSlots <= t < s - \perasBlockMinSlots

(It would then probably also need to be renamed.)

Reply:
> We need at the very least `\perasBlockMaxSlots > \perasBlockMinSlots`. Or maybe is this line suggesting that if `\perasBlockMinSlots = 30`, then `\perasBlockMaxSlots = 90`? That sounds too small, because there is only a 95% chance (with asc = 1/20) that there is a block in 60 slots, so in 5% of the cases we would not be allowed to vote in a round.

This value is indeed incorrect, see https://github.com/tweag/cardano-peras/pull/31#discussion_r1959544478

---

> Also, maybe it would be worth defining `\perasBlockMaxSlots` not from the first slot of the round but from `\perasBlockMinSlots`.

No strong opinion here; in general, protocol parameters can have constraints between each other (there even is a "guardrails script" that enforces some of these automatically).